File: JAVASCRIPT/Questões uri/1847.js
const pegarValoress = (line) => line.split(" ").map(a => +a)
const [a,b,c] = pegarValoress(lines.shift())
if (a>b && c>=b){
  console.log(":)")
}
else if (a<b && b>=c){
  console.log(":(")
}
else if (a<b && ((b-a)>(c-b))){
  console.log(":(")
}
else if (a<b && ((c-b)>=(b-a))){
  console.log(":)")
}
else if (a>b && ((a-b)>(b-c))){
  console.log(":)")
}
else if (a>b && ((a-b)<=(b-c))){
  console.log(":(")
}
else if (a==b && b<c){
  console.log(":)")
}
else if (a==b && b>c){
  console.log(":(")
}
else if (a==b && a==c && b==c){
  console.log(":(")
}

File: ESTUDO DE LINGUAGENS/PYTHON/Questões Uri/1847.py
a,b,c=map(int, input().split(" "))

if a>b and c>=b:
  print(":)")
elif a<b and b>=c:
  print(":(")
elif a<b and ((b-a)>(c-b)):
  print(":(")
elif a<b and ((c-b)>=(b-a)):
  print(":)")
elif a>b and ((a-b)>(b-c)):
  print(":)")
elif a>b and ((a-b)<=(b-c)):
  print(":(")
elif a==b and b<c:
  print(":)")
elif a==b and b>c:
  print(":(")
elif a==b and a==c and b==c:
  print(":(")